src/Resources/contao/controllers/BackendMain.php

```diff
@@ -113,7 +113,22 @@ public function run()
 		// Open a module
 		elseif (\Input::get('do'))
 		{
-			$this->Template->main .= $this->getBackendModule(\Input::get('do'));
+			$picker = null;
+
+			if (isset($_GET['picker']))
```

Comment:
Using `isset($_GET)` cannot be "overwritten" via `Input::setGet()`. I know that I have been doing it, too – maybe we should discuss it and be consistent?